docs/step3.md

```diff
@@ -42,22 +42,22 @@ As you can see, you have the result of the code that was run in that cell, as we
 
 * When you save a notebook, the code as well as the results are saved! So you can always reopen it to look at the results without having to run all the program again, while still having access to the code.
 
-Therefore the name **Notebook**: it's exactly like if you were taking notes about your experiments, what you will do, with which parametets (this is the code part), and what the experiment resulted in (this is the result from a cell once it's run). Along of course with all the notes you want to take (to do that, switch the cell type from the menu from `Code`to `Markup`).
+Notebooks are so names because they are just like a physical **Notebook**: it's exactly like if you were taking notes about your experiments, (which you will do) along with the code itself, including any parameters you set. You see the output of the experiment inline (this is the result from a cell once it's run), along with all the notes you want to take (to do that, switch the cell type from the menu from `Code` to `Markup`).
```

Comment:
Notebooks are so named
(and not "names")

Reply:
+ good catch, updated